client/components/plans/plan-actions/index.jsx

```diff
@@ -189,14 +229,14 @@ module.exports = React.createClass( {
 			);
 		}
 
-		if ( isFreePlan( this.props.plan ) ) {
+		if ( this.shouldOfferFreeTrial() ) {
 			return (
-				<div onClick={ this.handleAddToCart.bind( null, null, 'button' ) } className={ classes } />
+				<div onClick={ this.handleClickStartTrial } className={ classes } />
```

Comment:
Yep we could do that. I do think that avoiding bind here makes the code easier to understand / cleaner.